tools/bitfields.lisp

```diff
@@ -72,6 +72,22 @@
    "MAP_INVALIDATE_RANGE_BIT"
    "MAP_WRITE_BIT"
    "MAP_READ_BIT"))
+
+
+ (:enum-type

```

Comment:
Oops, forgot that part when merging. Looks like that file is unused, and correct place to fix it is in `tools/special-cases.lisp`. Fixed now.